core/src/main/scala/org/apache/spark/mapred/SparkHadoopMapRedUtil.scala

```diff
@@ -80,7 +81,7 @@ object SparkHadoopMapRedUtil extends Logging {
           logInfo(message)
           // We need to abort the task so that the driver can reschedule new attempts, if necessary
           committer.abortTask(mrTaskContext)
-          throw new CommitDeniedException(message, jobId, splitId, taskAttemptNumber)
+          throw new CommitDeniedException(message, ctx.stageId(), splitId, ctx.attemptNumber())
```

Comment:
hmm, shall we still keep `jobId` here?

Reply:
2.2 and later use the stage ID; I think it's more correct since that's what the coordinator uses, but don't really mind either way.